networks/local/Makefile

```diff
@@ -0,0 +1,6 @@
+# Makefile for the "desmosd" docker image.
+
+all:
+	docker build --tag desmos-labs/desmosdnode desmosdnode
```

Comment:
The organization I created on Docker Hub is `desmoslabs` (https://hub.docker.com/orgs/desmoslabs). I think the tag has to be `desmoslabs/desmosdnode` here.

Reply:
Fixed it!